File: tools/sms/src/main/java/top/bettercode/summer/tools/sms/SmsSysException.java
package top.bettercode.summer.tools.sms;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

/**
 * @author Peter Wu
 */
@ResponseStatus(HttpStatus.BAD_REQUEST)
public class SmsSysException extends IllegalArgumentException {

  private static final long serialVersionUID = 1L;

  public SmsSysException(String message) {
    super("短信平台：" + message);
  }
}

File: tools/sms/src/main/kotlin/top/bettercode/summer/tools/sms/SmsSysException.kt
package top.bettercode.summer.tools.sms

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

/**
 * @author Peter Wu
 */
@ResponseStatus(HttpStatus.BAD_REQUEST)
class SmsSysException(message: String) : IllegalArgumentException("短信平台：$message") {
}
